borgbackup: 1.2.1 -> 1.2.2

kilianar opened the pull request:
###### Description of changes
https://github.com/borgbackup/borg/releases/tag/1.2.2

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

kilianar committed:
borgbackup: 1.2.1 -> 1.2.2

https://github.com/borgbackup/borg/releases/tag/1.2.2

anthonyroussel commented:
Result of `nixpkgs-review pr 187638` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>borgbackup</li>
    <li>borgmatic</li>
    <li>pika-backup</li>
    <li>vorta</li>
  </ul>
</details>

mweinelt committed:
borgbackup: move manpages into man output

By creating the man output the multiple-outputs hook will automatically
move manpages in the correct path into it.

SuperSandro2000 commented:
Closing in favor of #187651 because that also adds the man output.

dotlambda commented:
> Closing in favor of #187651 because that also adds the man output.

That's not very nice. You could just cherry-pick that commit onto this branch.

SuperSandro2000 commented:
> You could just cherry-pick that commit onto this branch.

Its just that pushing into a PR where I do not have the remote in my git repository usually takes a few minutes and sometimes just sits there and never finishes. Since The changes are identically I've chosen the faster way for me.
